VERSION

```diff
@@ -1 +1 @@
-1.0.0
\ No newline at end of file
+1.1.0
```

Comment:
bump to 1.2.0 since we already at 1.1.0 right now

```suggestion
1.2.0
```

Reply:
bumped to v1.2.0